test/pantries.js

```diff
@@ -1,4 +1,5 @@
 process.env.NODE_ENV = 'test';
```

Comment:
never used Chai before but this is really straight forward. Thanks for introducing it.